src/mudblazor/MudBlazor.Template/wwwroot/app.css

```diff
@@ -1,20 +1,53 @@
-/*
-	This CSS file matches the color scheme from MudBlazor to Bootstrap when utilized for authentication.
-	The file remains available at all times for demonstration purposes,
-	but it is exclusively employed in the 'App.razor' component when authentication is enabled.
-*/
+/*#if (SampleContent)*/
```

Comment:
This entire file could probably be empty if I upgrade all of the Auth pages to use MudBlazor components.  The thing is, it's 20+ pages, which means any future efforts to rebuild templates from scratch will take about a week to get through.